lib/__tests__/cloudworker.test.js

```diff
@@ -165,4 +165,14 @@ describe('cloudworker', () => {
     expect(res.headers.get('x-colo')).toEqual('YYZ')
     done()
   })
+
+  test('throws on eval', async () => {
+    const bindings = { EvalError: EvalError }
+    expect(() => { new Cloudworker(`eval('true')`, { bindings }) }).toThrow(new EvalError('Code generation from strings disallowed for this context')) // eslint-disable-line no-new
+  })
+
+  test('throws on new Function(<string>)', async () => {
+    const bindings = { EvalError: EvalError }
+    expect(() => { new Cloudworker(`new Function('true')`, { bindings }) }).toThrow(new EvalError('Code generation from strings disallowed for this context')) // eslint-disable-line no-new
+  })
 })
```

Comment:
could you please add a test for disallowing `Function.constructor("console.log(1)")()`?

Also note that code generation applies to wasm as well:
```js
  const arr = [];
  new WebAssembly.Module(arr)
```
should be disallowed